src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
+  - Fully exited validators will automatically receive their remaining balance
```

Comment:
"Fully exited validators" feels like a somewhat confusing term...

Maybe something like "Validators who fully exit staking will automatically receive their remaining balance"?

Reply:
We may want to ditch the "automatically" party here as well

```suggestion
  - Validators who fully exit staking will receive their remaining balance
```

By removing "automatic" I think it reduces confusion around "will a full withdrawal _happen to me automatically?_", which the answer is of course no (except for slashing/low balance)... the exit must be initiated but the rest is automatic. I think the above wording reduces the chances of somebody thinking that.